package/MDAnalysis/transformations/__init__.py

```diff
@@ -69,14 +46,50 @@ def __call__(self, ts):
             #  apply changes to the Timestep object
             return ts
 
+See `MDAnalysis.transformations.translate` for a simple example.
+
+These transformation functions can be called by the user for any given timestep
+of the trajectory, added as a workflow using :meth:`add_transformations`
+of the :mod:`~MDAnalysis.coordinates.base`, or upon Universe creation using
+the keyword argument `transformations`. Note that in the two latter cases, the
+workflow cannot be changed after being defined. for example:
+
+.. code-block:: python
+
+    u = mda.Universe(GRO, XTC)
+    ts = u.trajectory[0]
+    trans = transformation(args)
+    ts = trans(ts)
+
+    #  or add as a workflow
+    u.trajectory.add_transformations(trans)
+
+Transformations can also be created as a closure/nested function.
+In addition to the specific arguments that each transformation can take, they
+also contain a wrapped function that takes a `Timestep` object as argument.
+So, a closure-style transformation can be roughly defined as follows:
+
+.. code-block:: python
+
+    def transformation(*args,**kwargs):
+        # do some things
+            def wrapped(ts):
+                # apply changes to the Timestep object
+                return ts
+
+            return wrapped
 
-Note it does not mean that the old closure/wrapped function implementation will fail.
-They can still be used if one does not need serialization, which is a prerequisite for parallel
-analysis.
+Note, to meet the need of serialization of universe, only transformation class
+are used after MDAnlaysis 2.0.0. One can still write functions (closures) as in
+MDA 1.x, but that these cannot be serialized and thus will not work with all
+forms of parallel analysis. For detailed descriptions about how to write a
+closure-style transformation, read the code in MDA 1.x as a reference
+or read MDAnalysis UserGuide.
```

Comment:
```suggestion

.. Note::
   Although functions (closures) work as transformations, they are not used in
   in MDAnalysis from release 2.0.0 onwards because they cannot be reliably
   serialized and thus a :class:`Universe` with such transformations cannot be 
   used with common parallelization schemes (e.g., ones based on 
   :mod:`multiprocessing`). 
   For detailed descriptions about how to write a closure-style transformation, 
   please refer to MDAnalysis 1.x documentation.
```